### ISSUE TITLE
Option to disable the home button

### ISSUE THREAD

**Issue by Madis0** [fork-maintainers/iceraven-browser#437]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have exactly one feature request described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Feature Requests are better when they’re open-ended instead of demanding a specific solution e.g: “I want an easier way to do X” instead of “add Y”)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

### What is the user problem or growth opportunity you want to see solved?
More space for the URL, less clutter in the UI.

### How do you know that this problem exists today? Why is this important?
This is important because Mozilla added a home button, but not the option to disable it. Lack of an option makes it worse than Chromium alternatives and the desktop browsers (Firefox and others).

### Who will benefit from it?
People who prefer minimalism.

See also: #53 for the opposite request.

**Comment by interfect** [comment]
Hmm. This does sound like a good thing to have. Any clue how this could be done in a way that won't give rise to a jillion merge conflicts?

**Comment by Madis0** [comment]
An applied git patch perhaps, instead of rebasing on the change? I don't know how Fennec or its development actually works though.

**Comment by seniordevel** [comment]
This can easily be done by just hiding this button. It will not cause any problem. 

Just a few lines of code would be needed, and some more to add a setting in about:config.

I would appreciate it.

**Comment by interfect** [comment]
If anyone wants to write a PR for this, that would be great. I'm not sure if `about:config` would be the right place for it, since that lives in Android Components, and since we've already added stuff to the normal settings screens successfully. Though with the monorepo merge maybe we'll end up building Android Components and thus able to patch it.

**Comment by seniordevel** [comment]
> we've already added stuff to the normal settings screens successfully

Adding to the settings screen would be fine as well.  I suggested about:config  for just one reason: This is quickly done - whereas working with GUI is much more effort. 

Maybe I'm wrong, but for me it seems the dev team here is small and their time is limited.

**Comment by bt4ibwem8** [comment]
I can see we still cannot get rid of this button. Is it technically hard to add hide option in the settings?

**Comment by interfect** [comment]
Probably not. Would you like to work out how the address bar view stuff
works and figure out what the setting would need to do to get it to be
conditionally visible?

On Sunday, February 4, 2024, bt4ibwem8 ***@***.***> wrote:
> I can see we still cannot get rid of this button. Is it technically hard
to add hide option in the settings?
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you commented.Message ID:
***@***.***>

**Comment by bt4ibwem8** [comment]
> to add hide option in the settings?

correct. If it is not hard why it is not the option just to tick the checkbox in the settings? This button just clutter interface. Even Chrome does allow to hide it easily...

**Comment by interfect** [comment]
It's not that it would be *hard* to do, it's just that nobody has done it
yet.

**Comment by akliuxingyuan** [comment]
Just notice this issue, hope not too late 😄
